Asterix/wfsc/estimator.py

```diff
@@ -263,7 +263,7 @@ def estimate(self, testbed: Testbed, entrance_EF=1., voltage_vector=0., perfect_
             estimation of the Electrical field
         """
 
-        if 'wavelength' in kwargs:
+        if 'wavelength' in kwargs.keys():
```

Comment:
ok, I did not know that. I have not removed all the "keys" in asterix since some are deeply embedded in very important things of the code (the stuff that concatenate the optical systems), but I have modified the ones I added in this PR

Reply:
Perfect. I think that's the way to go, no need to fix something in this PR that was not the main intention.